notebook/color-notebook.js

```diff
@@ -202,7 +203,7 @@ export default class Notebook {
 		}
 		catch (e) {
 			// Syntax error
-			var error = e;
```

Comment:
I feel like there may have been a reason this used `var`. let me know if we cant make this change